### PR TITLE
Correct for PM on the target editor's elevation plot

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: explore/src/main/scala/explore/tabs/TargetTabContents.scala [hunk 1]
```diff
@@ -167,6 +167,13 @@ object TargetTabContents extends TwoPanels:
         )
     )
 
+    def getObsTime(idsToEdit: ObsIdSet): ProgramSummaries => Option[Instant] = a =>
```
Comment (rpiaggio) [inline explore/src/main/scala/explore/tabs/TargetTabContents.scala:170]: Why move this here? It doesn't seem to be called from anywhere else.

**Review by rpiaggio — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/tabs/TargetTabContents.scala (hunk 1 above)
Comment (cquiroz) [reply explore/src/main/scala/explore/tabs/TargetTabContents.scala:170]: I was going to share it with another function but I didn't need it after all

File: explore/src/main/scala/explore/tabs/TargetTabContents.scala [hunk 2]
```diff
@@ -391,7 +394,6 @@ object TargetTabContents extends TwoPanels:
                 props.userId,
                 targetId.some,
                 none,
-                // TODO PM correct the coordinates
                 CoordinatesAtVizTime(Target.Sidereal.baseCoordinates.get(target.get)).some,
```
Comment (toddburnside) [inline explore/src/main/scala/explore/tabs/TargetTabContents.scala:397]: Are these PM corrected?

Comment (cquiroz) [reply explore/src/main/scala/explore/tabs/TargetTabContents.scala:397]: There is no time in this case, we can only fallback to base

**Pull request merged**